### DISCUSSION TITLE
Аналог tpws для windows

### DISCUSSION THREAD

**Discussion by testing765** [bol-van/zapret#829] [General]
Не планируется ли какой то аналог tpws для windows? Чтобы запускать как проксю без windivert и костылей вроде WSL?

**Comment by bol-van** [comment]
Для этого есть препятствия.
tpws написан с epoll, которого нет под виндой. Придется переписывать значительную часть кода.
Под виндой не работает disorder.  oob и mss  под вопросом. Под WSL они точно не работают.

**Reply by bol-van** [reply]
Рекомендую использовать byedpi
